### ISSUE TITLE
Add FadingOption to disable all code fading.

### ISSUE THREAD

**Issue by JoeRobich** [dotnet/roslyn#58400]
This option would disable the unnecessary code tagger.

**Comment by ryzngard** [comment]
Looking at using this for our preview windows to help with alpha issues related to red/green backgrounds. With meeting minimum contrast rating in the normal editor we can't also guarantee all colors are CCR compliant with red/green backgrounds.

**Comment by sharwell** [comment]
@ryzngard Fading can be helpful in previews as well, so hoping there is an option to disable the workaround you're looking at

**Comment by ryzngard** [comment]
> @ryzngard Fading can be helpful in previews as well, so hoping there is an option to disable the workaround you're looking at

Agreed. We're looking at ways we can avoid having to do this. There are cases where accessibility requirements do degrade user experience for all parties, and we'd like to keep functionality present. This is one of the routes we're looking at taking.

**Comment by jasonmalinowski** [comment]
Conclusions:

1. We're still hoping to have an accessibility exception for this which renders any conversation here moot.
2. We're also looking at potentially changing the "red" background color.

**Comment by ryzngard** [comment]
After review with accessibility team we will be looking at keeping this, with an improved option that would be accessible to users that are negatively impacted by the fade. This would be an option and provide an alternative visual cue instead of fading for the same information.

**Comment by ryzngard** [comment]
@JoeRobich let's use this to track improvements to unused code visual indicators. Assigning to you to drive design in this area

**Comment by CyrusNajmabadi** [comment]
Closing as speculative.  If this needs to happen, it needs to go in a milestone.
